content/departments/cloud/technical-docs/operations.md

```diff
@@ -374,6 +390,8 @@ Once you have identified a repo is constantly failing to be updated/fetched, exe
    export DEPLOYMENT=$(gcloud compute instances list --project "$PROJECT_PREFIX-$CUSTOMER" | grep -v "executors" | awk 'NR>1 { if ($1 ~ "-red-") print "red"; else print "black"; }')
    ```
 
+````
```

Comment:
Looks like an oversight? i.e. should we delete it

Reply:
My formatter must have gone a bit haywire